dockertest/images.py

```diff
@@ -92,6 +92,10 @@ def __init__(self, repo, tag, long_id, created, size,
         self.user = user
 
         self.short_id = long_id[:12]
+        # docker-1.10 output includes a prefix describing the hash type
+        # FIXME: find canonical list
+        if long_id.startswith("sha256:"):
+            self.short_id = long_id[7:19]
```

Comment:
Ugg, what a PITA.  Knowing Docker, it'll probably change again in 6-mo.  It's almost as if we need a separate class just to handle the stupid ID.